database/tables/group_user_roles.pg

```diff
@@ -0,0 +1,15 @@
+columns
+    group_id: bigint not null
+    group_role_id: bigint not null
+    user_id: bigint not null
+
+indexes
+    group_user_roles_pkey: PRIMARY KEY (group_id, user_id, group_role_id) USING btree (group_id, user_id, group_role_id)
```

Comment:
I see he mentioned that `assessment_question_role_permissions` and `group_users` are also issues. Do you want us to fix these as well, or leave them since they've already been merged?

Reply:
Unfortunately I don't see a clear path to doing so in a way that can tolerate zero-downtime rolling deploys 😢 I think it's fine to leave this alone.